Teacher Workout/Features/Onboarding/View/OnboardingViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewController(_ viewController: OnboardingViewController, didSelectPageIndex index: Int)
+    func onboardingViewControllerDidSkip(_ viewController: OnboardingViewController)
+    func onboardingViewControllerDidPressNext(_ viewController: OnboardingViewController, currentIndex index: Int)
+}
+
+class OnboardingViewController: UIViewController, OnboardingHeroViewDelegate {
+    @IBOutlet var onboardingHeroView: OnboardingHeroView!
+    @IBOutlet var onboardingTitleLabel: UILabel!
+    @IBOutlet var onboardingDescription: UILabel!
+    @IBOutlet var nextButton: UIButton!
+
+    var heroDelegate: OnboardingHeroViewDelegate?
+    var delegate: OnboardingViewControllerDelegate?
+
+    var viewModel: OnboardingViewModel!
+
+    var currentPageIndex: Int = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        onboardingHeroView.delegate = self
+        setupUI()
+    }
+
+    private func setupUI() {
+        setupHeroView()
+        onboardingTitleLabel.text = viewModel.pageTitle
+        onboardingDescription.text = viewModel.pageDescription
+        nextButton.contentEdgeInsets = UIEdgeInsets(top: 16, left: 20, bottom: 16, right: 20)
+        nextButton.layer.cornerRadius = 24
+        nextButton.layer.borderWidth = 0.5
+        nextButton.layer.borderColor = UIColor(red: 0.031, green: 0.569, blue: 0.698, alpha: 1).cgColor
```

Comment:
Use colors defined on Assets only. Easier for us to change the theme in the future.